docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -780,13 +780,45 @@ Specific observer outputs and their stratifications may vary by model run as nee
     - ``/mnt/team/simulation_science/pub/models/vivarium_gates_mncnh/results/raw_csmr/``
     - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
   * - 6.1
-    - Rerun LBWSG PAF calculation for Ethiopia
+    - Rerun with LBWSG PAF changes for Ethiopia: (1) fix sex-specificity bug in LBWSG PAF calculation, and (2) use LBWSG exposure at birth for calculation of the ENN LBWSG PAF
+    - Complete
+    - 10
+    - 100,000
+    - All scenarios
+    - ``mnt/team/simulation_science/pub/models/vivarium_gates_mncnh/results/model6.1``
+    - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 50,000 simulants = 500,000 total population.
+  * - 6.2
+    - Same specifications as model 6.1, but this time with the exponential rate-to-probability conversion (:math:`p= 1 - e^(-\text{rate} * \text{duration scaling factor}))` in `this function <https://github.com/ihmeuw/vivarium_gates_mncnh/blob/29fe810c2f1abf5b358a452d3f59ffdda266afe8/src/vivarium_gates_mncnh/utilities.py#L187-L193>`_
```

Comment:
```suggestion
    - Same specifications as model 6.1, but this time with the exponential rate-to-probability conversion (:math:`p= 1 - e^{(-\text{rate} * \text{duration scaling factor})}`) in `this function <https://github.com/ihmeuw/vivarium_gates_mncnh/blob/29fe810c2f1abf5b358a452d3f59ffdda266afe8/src/vivarium_gates_mncnh/utilities.py#L187-L193>`_
```
You need braces to put the whole expression in the exponent. Also, I think the backtick was in the wrong place.